spec/.eslintrc.js

```diff
@@ -12,5 +12,9 @@ module.exports = {
 		chai: false,
 		sinon: false,
 		Hand: false,
+		touchEventType: false, /* defined in SpecHelper.js */
+		createContainer: false, /* defined in SpecHelper.js */
+		removeMapContainer: false, /* defined in SpecHelper.js */
+		pointerType: false /* defined in SpecHelper.js */
```

Comment:
These should no longer be needed, as `SpecHelper.js` is now a module, and doesn't expose any globals.
```suggestion
```